### ISSUE TITLE
bug: build failures due to BoS keys

### ISSUE THREAD

**Issue by qdzlug** [nginxinc/kic-reference-architectures#162]
**Describe the bug**
When running our automated builds we are receiving the following error:

```
[2022-06-15T17:27:03.048Z] #=============================================================================#

[2022-06-15T17:27:03.048Z] #     _  __          _                                      __   _            #

[2022-06-15T17:27:03.048Z] #    | |/ /  _   _  | |__     ___    ___    ___    _ __    / _| (_)   __ _    #

[2022-06-15T17:27:03.048Z] #    | ' /  | | | | | '_ \   / _ \  / __|  / _ \  | '_ \  | |_  | |  / _` |   #

[2022-06-15T17:27:03.048Z] #    | . \  | |_| | | |_) | |  __/ | (__  | (_) | | | | | |  _| | | | (_| |   #

[2022-06-15T17:27:03.048Z] #    |_|\_\  \__,_| |_.__/   \___|  \___|  \___/  |_| |_| |_|   |_|  \__, |   #

[2022-06-15T17:27:03.048Z] #                                                                    |___/    #

[2022-06-15T17:27:03.048Z] #                                                                             #

[2022-06-15T17:27:03.048Z] #=============================================================================#

[2022-06-15T17:27:03.048Z] 

[2022-06-15T17:27:03.048Z] 

[2022-06-15T17:27:03.307Z] Previewing update (marajenkmkube27)

[2022-06-15T17:27:03.307Z] 

[2022-06-15T17:27:03.307Z] View Live: https://app.pulumi.com/qdzlug/kubeconfig/marajenkmkube27/previews/bc1e99bb-9006-450f-84c6-d8a2278df6c9

[2022-06-15T17:27:03.307Z] 

[2022-06-15T17:27:03.564Z]  

[2022-06-15T17:27:03.564Z] error: failed to decrypt encrypted configuration value 'sirius:accounts_pwd': [400] Message authentication failed

[2022-06-15T17:27:03.564Z] This can occur when a secret is copied from one stack to another. Encryption of secrets is done per-stack and

[2022-06-15T17:27:03.564Z] it is not possible to share an encrypted configuration value across stacks.

[2022-06-15T17:27:03.564Z] 

[2022-06-15T17:27:03.564Z] You can re-encrypt your configuration by running 'pulumi config set sirius:accounts_pwd [value] --secret' with your

[2022-06-15T17:27:03.564Z] new stack selected.

[2022-06-15T17:27:03.564Z] 

[2022-06-15T17:27:03.564Z] refusing to proceed

```

**To Reproduce**
1. Test deploy with Jenkins
2. Error will be displayed at kubeconfig project

**Expected behavior**
Should work.

**Your envineronment**
* n/a

**Additional context**
None,

**Comment by qdzlug** [comment]
This is almost certainly an issue with pathing for the configuration (using the wrong pulumi directory). Will fix.

**Comment by qdzlug** [comment]
Will be closed by #167.
